pkg/controllers/nodeclaim/termination/suite_test.go

```diff
@@ -194,6 +212,9 @@ var _ = Describe("Termination", func() {
 		// Expect the nodeClaim to be gone
 		Expect(env.Client.Delete(ctx, nodeClaim)).To(Succeed())
 		ExpectReconcileSucceeded(ctx, nodeClaimTerminationController, client.ObjectKeyFromObject(nodeClaim))
+		nodeClaim.StatusConditions().MarkTrue(v1beta1.Terminating)
```

Comment:
If you mark the nodeClaim as terminateing, then we would not make a delete call. Should we be setting the status condition here?
```suggestion
```